flink-connector-bigquery/pom.xml

```diff
@@ -33,6 +33,12 @@ under the License.
     <packaging>jar</packaging>
 
     <dependencies>
+		<!-- joda dependency -->
```

Comment:
This should go under `utilities`: https://github.com/GoogleCloudDataproc/flink-bigquery-connector/blob/154aeb0f419a2aa5128df0eb91e05e1e9f86e581/flink-connector-bigquery/pom.xml#L72

Reply:
Fixed.